libparsec/crates/platform_mountpoint/src/unix/filesystem.rs

```diff
@@ -208,6 +208,20 @@ pub(crate) static LOOKUP_HOOK: Mutex<
     Option<Box<dyn FnMut(&FsPath) -> Option<Result<EntryStat, WorkspaceStatEntryError>> + Send>>,
 > = Mutex::new(None);
 
+macro_rules! add_capabilities {
```

Comment:
I'd add a comment
```suggestion
/// Helper macro to add one or many capabilities to fuser config.
macro_rules! add_capabilities {
```
Or maybe even with an example?
```suggestion
/// Helper macro to add one (or multiple) capabilities to fuser config.
///
/// Example:
/// ```
/// add_capabilities!(config, <capability>)
/// add_capabilities!(config, <capability1>, ..., <capabilityN>)
/// ```
macro_rules! add_capabilities {
```